src/components/shared/Filter/ActivityFilter/index.scss

```diff
@@ -17,3 +17,28 @@
 .select-filter {
   padding-top: 15px;
 }
+
+.sg-date-picker {
+  margin-top: 0.5rem;
+}
+
+.CalendarDay__selected_span {
+  background: #1a82e2;
```

Comment:
we can place #1a82e2 into a variable like $some_color name since it's reused throughout